queuelib/queue.py

```diff
@@ -37,12 +37,14 @@ class FifoDiskQueue(object):
     szhdr_format = ">L"
     szhdr_size = struct.calcsize(szhdr_format)
 
-    def __init__(self, path, chunksize=100000):
+    def __init__(self, path, chunksize=100000, filesize=1000000, syncAll=False):
```

Comment:
Could we make this change backward-compatible? (i.e. `filesize=None`, and handle it appropriately later)